resources/lang/en/people.php

```diff
@@ -10,28 +10,28 @@
 
     //index
     'people_not_found' => 'Contact not found',
-    'people_list_number_kids' => '1 kid|:count kids',
+    'people_list_number_kids' => 'One child|:count children',
```

Comment:
I don't see any reason not to keep the number even for "1"
```suggestion
    'people_list_number_kids' => ':count child|:count children',
```

Reply:
Most English style guides recommend using words for numbers below ten, and especially when they start a sentence.  I don’t know how to implement that for two through nine, so I split the difference and did it with ‘one’.

Reply:
In this case, it's not really a "sentence", but just displayed information.
There are 2 reasons not to change that:
- consistency in the display: if there is 1 or 2 items, I don't want to display it differently. The important part is the number of children. It's easier to see the number of children in '1 child' than in 'One child'.
- translation rules: It's hard to make the translation, and so the contributors in translation, know all the programmatic rules. All languages does not follow english here ('1' for singular version, '0' and '2' or more for plural version), see this [laravel function](https://github.com/laravel/framework/blob/8.x/src/Illuminate/Translation/MessageSelector.php#L110), and I'm afraid all of them will translate that following "One child" as a text, and ":count children" as another text, which is not the case.